src/components/DeploymentsList.vue

```diff
@@ -26,11 +26,11 @@
 
       <template #deployment-name="{ row }">
         <div class="deployment-list__name-col">
-          <span>
-            <p-link :to="routes.deployment(row.id)" class="deployment-list__name">
+          <span class="deployment-list__name">
+            <p-link :to="routes.deployment(row.id)">
               {{ row.name }}
-              <DeploymentStatusIcon v-if="can.access.deploymentStatus" :status="row.status" />
             </p-link>
+            <DeploymentStatusIcon :status="row.status" />
```

Comment:
Previously the icon was just inlined up against the name inside of a link. Now the icon is outside of the anchor tag (since it's a tooltip target also). It also looks like there was an intended `mr-2` to put 4px of space between, but since the icon is a _child_ of `<p-link>` there was just "implicit" inline spacing and now the flex items-center lifts the icon up just a tad to vertically center it.